<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageAnimalsView</class>
 <widget class="QMainWindow" name="ManageAnimalsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>546</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>881</width>
    <height>546</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(232, 210, 184);
font: 10pt &quot;URW Gothic L&quot;;</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="leftLayout" stretch="0,0,0,0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="animalTitleLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Noto Sans Devanagari UI</family>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 15pt &quot;Noto Sans Devanagari UI&quot;;
font: bold</string>
        </property>
        <property name="text">
         <string>Animals</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="animalTableView">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>502</width>
          <height>330</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>516</width>
          <height>330</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editButton">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="titleDetailsLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Noto Sans Devanagari UI</family>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 15pt &quot;Noto Sans Devanagari UI&quot;;
font: bold</string>
        </property>
        <property name="text">
         <string>Details</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>350</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>400</height>
         </size>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="basicInfo">
         <attribute name="title">
          <string>Basic Info</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>351</width>
            <height>231</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="animalDetailsLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::FieldsStayAtSizeHint</enum>
           </property>
           <property name="horizontalSpacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="idLabel">
             <property name="text">
              <string>ID: </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="displayID">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="nameLabel">
             <property name="text">
              <string>Name: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="displayName">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="ageLabel">
             <property name="text">
              <string>Age: </string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="displayAge">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="genderLabel">
             <property name="text">
              <string>Gender: </string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="displayGender">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="typeLabel">
             <property name="text">
              <string>Type: </string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="displayType">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Personality</string>
         </attribute>
         <widget class="QWidget" name="formLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>351</width>
            <height>311</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout_12">
           <item row="0" column="0">
            <widget class="QLabel" name="timidityLabel">
             <property name="text">
              <string>Timidity:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="sociabilityLabel">
             <property name="text">
              <string>Sociability:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="aggressionLabel">
             <property name="text">
              <string>Aggression:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="obedienceLabel">
             <property name="text">
              <string>Obedience:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="loyaltyLabel">
             <property name="text">
              <string>Loyalty:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="affectionLabel">
             <property name="text">
              <string>Affection:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="activityLabel">
             <property name="text">
              <string>Activity:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="excitabilityLabel">
             <property name="text">
              <string>Excitability:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="expressivenessLabel">
             <property name="text">
              <string>Expressiveness:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="displayTimidity">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="displaySociability">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="displayAggression">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="displayObedience">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="displayLoyalty">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="displayAffection">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="displayActivity">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLabel" name="displayExcitability">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLabel" name="displayExpressiveness">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Other</string>
         </attribute>
         <widget class="QWidget" name="formLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>341</width>
            <height>311</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout_13">
           <item row="0" column="0">
            <widget class="QLabel" name="sizeLabel">
             <property name="text">
              <string>Size:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="sheddingLabel">
             <property name="text">
              <string>Shedding:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="trainingSuitabilityLabel">
             <property name="text">
              <string>Training Suitability:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="currTrainingLabel">
             <property name="text">
              <string>Current Training:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="intelligenceLabel">
             <property name="text">
              <string>Intelligence:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="costLabel">
             <property name="text">
              <string>Cost:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="timeLabel">
             <property name="text">
              <string>Time Commitment:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Apartment-Living:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Kid Friendliness:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="displaySize">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="displayShedding">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="displayTrainingSuitability">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="displayCurrTraining">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="displayIntelligence">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="displayCost">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="displayTime">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLabel" name="displayAptCompatibility">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLabel" name="displayKidFriendliness">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>180</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
